apps/coscrad-frontend-e2e/src/e2e/resources/digital-texts/admin/digital-text.publication.e2e.cy.ts

```diff
@@ -0,0 +1,89 @@
+import { AggregateType, ResourceType } from '@coscrad/api-interfaces';
+import { buildDummyAggregateCompositeIdentifier } from '../../../../support/utilities';
+
+const unpublishedDigitalTextCompositeIdentifier = buildDummyAggregateCompositeIdentifier(
+    AggregateType.digitalText,
+    1
+);
+
+const buildDetailRoute = (digitalTextId: string) => `/Resources/DigitalTexts/${digitalTextId}`;
+
+describe(`Digital Text publication`, () => {
+    before(() => {
+        cy.clearDatabase();
+
+        cy.seedTestUuids(100);
+
+        cy.executeCommandStreamByName('users:create-admin');
+    });
+
+    describe(`when the digital text is not already published`, () => {
+        before(() => {
+            cy.seedDataWithCommand('CREATE_DIGITAL_TEXT', {
+                aggregateCompositeIdentifier: unpublishedDigitalTextCompositeIdentifier,
+            });
+        });
+
+        // TODO Should we move this to the detail query flow test?
+        describe(`when the user is not authenticated`, () => {
```

Comment:
We also need to test that when the user is not authenticated, no command buttons are available. This applies to every command. It is an issue of UX not security as the back-end will reject the command with a not authorized either way.